addons/binding/org.openhab.binding.bluetooth.ruuvitag/ESH-INF/thing/ruuvitag.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ruuvitag_beacon">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bluegiga" />
+			<bridge-type-ref id="bluez" />
+		</supported-bridge-type-refs>
+
+		<label>RuuviTag SmartBeacon</label>
+		<description>A RuuviTag SmartBeacon</description>
+
+		<channels>
+			<channel id="rssi" typeId="rssi" />
+
+			<channel id="accelerationx" typeId="ruuvitag_accelerationx" />
+			<channel id="accelerationy" typeId="ruuvitag_accelerationy" />
+			<channel id="accelerationz" typeId="ruuvitag_accelerationz" />
+			<channel id="battery_voltage" typeId="ruuvitag_battery_voltage" />
+			<channel id="data_format" typeId="ruuvitag_data_format" />
+			<channel id="humidity" typeId="ruuvitag_humidity" />
+			<channel id="measurement_sequence_number" typeId="ruuvitag_measurement_sequence_number" />
+			<channel id="movement_counter" typeId="ruuvitag_movement_counter" />
+			<channel id="pressure" typeId="ruuvitag_pressure" />
+			<channel id="temperature" typeId="ruuvitag_temperature" />
+			<channel id="tx_power" typeId="ruuvitag_tx_power" />
+		</channels>
+
+		<config-description>
+			<parameter name="address" type="text">
+				<label>Address</label>
+				<description>Bluetooth address in XX:XX:XX:XX:XX:XX format</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="ruuvitag_accelerationx">
+		<item-type>Number:Acceleration</item-type>
+		<label>Acceleration X</label>
+		<state readOnly="true" pattern="%.2f %unit%" />
```

Comment:
I would increase the precision to 3 as I had 0 values with 2 precision.
```suggestion
		<state readOnly="true" pattern="%.3f %unit%" />
```

Reply:
Fixed in eaa6957